### DISCUSSION TITLE
scss/sass to css compiler?

### DISCUSSION THREAD

**Discussion by aelbore** [swc-project/swc#4837] [Ideas]
Any plans to have scss to css compiler?

**Comment by kwonoj** [comment]
I'd personaly consider this is territory for the plugins. If upstream swc missing some features to support it as plugin, that may need to be implemented. (i.e we only have plugin support for transform only yet)

**Reply by aelbore** [reply]
maybe this is a suggestion or idea on plugins?

**Reply by jaw-sh** [reply]
This is a confusing answer because there is a merged ticket about CSS/SCSS support. #16

**Reply by kwonoj** [reply]
It's css, not scss/sass support so far I know. Issue itself included scss in the title but I believe we do not have those in place yet - it was closed for initial css support.

**Reply by aelbore** [reply]
i think it would really great if theres a plugin (in rust) to compile or transpile scss into css, im currently using `sass` now which is ruby base.

**Reply by kwonoj** [reply]
Can someone mind to elaborate if there's reason can't use de facto impl (dart-sass)? Afaik it's performant enough in most cases. If swc supports scss (via plugin), what's major difference?

**Reply by aelbore** [reply]
i think @jaw-sh said in the swc/plugins discussion it would be nice if you stay on the same ecosystem. but yeah you right also you can still use `sass` or `dart-sass` 
not sure if we can call or execute sass package inside rust plugin, if it is possible it really great
